From e094bf7ccbc29a48d23ffbc3f2c6357ba4c0b9e4 Mon Sep 17 00:00:00 2001
From: Masaharu Hayakawa <masaharu.hayakawa.ry@renesas.com>
Date: Wed, 21 Feb 2018 11:09:16 +0900
Subject: [PATCH 220/909] mmc: renesas_sdhi: Add r8a77990 to whitelist

commit b67612ee67df27a8b527593d2ae04bb427b3dbc3 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This patch adds R8A77990 to whitelist for DMAC.

I've successfully tested eMMC and SD card on E3 Ebisu board.
R8A77990 SoC does not have RX DMA errata.

Signed-off-by: Masaharu Hayakawa <masaharu.hayakawa.ry@renesas.com>
[saito: rebased on v4.14.75, add only soc_id.
        rework commit title and message. original title is
        "mmc: renesas_sdhi: Add r8a77990 support" ]
Signed-off-by: Takeshi Saito <takeshi.saito.xv@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/mmc/host/renesas_sdhi_internal_dmac.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/mmc/host/renesas_sdhi_internal_dmac.c b/drivers/mmc/host/renesas_sdhi_internal_dmac.c
index 646a23714763..9bab0d27fac1 100644
--- a/drivers/mmc/host/renesas_sdhi_internal_dmac.c
+++ b/drivers/mmc/host/renesas_sdhi_internal_dmac.c
@@ -319,6 +319,7 @@ static const struct soc_device_attribute gen3_soc_whitelist[] = {
 	{ .soc_id = "r8a7796" },
 	{ .soc_id = "r8a77965" },
 	{ .soc_id = "r8a77980" },
+	{ .soc_id = "r8a77990" },
 	{ .soc_id = "r8a77995" },
 	{ /* sentinel */ }
 };
-- 
2.17.1

